doc/source/Resources/pyaedt_installer_from_aedt.py

```diff
@@ -87,15 +87,13 @@ def run_pyinstaller_from_c_python(oDesktop):
         if version <= "231":
             f.write("from pyaedt.workflows.installer.pyaedt_installer import add_pyaedt_to_aedt\n")
             f.write(
-                'add_pyaedt_to_aedt(aedt_version="{}", personallib=r"{}")\n'.format(
-                    oDesktop.GetVersion()[:6], oDesktop.GetPersonalLibDirectory()))
+                f"'add_pyaedt_to_aedt(aedt_version=\"{oDesktop.GetVersion()[:6]}\", personallib=r\"{oDesktop.GetPersonalLibDirectory()}\")\n'")
         else:
             f.write("from ansys.aedt.core.workflows.installer.pyaedt_installer import add_pyaedt_to_aedt\n")
             f.write(
-                'add_pyaedt_to_aedt(aedt_version="{}", personal_lib=r"{}")\n'.format(
-                    oDesktop.GetVersion()[:6], oDesktop.GetPersonalLibDirectory()))
+                f"'add_pyaedt_to_aedt(aedt_version=\"{oDesktop.GetVersion()[:6]}\", personal_lib=r\"{oDesktop.GetPersonalLibDirectory()}\")\n'")
 
-    command = r'"{}" "{}"'.format(python_exe, python_script)
+    command = f"'\"{python_exe}\" \"{python_script}\"'"
```

Comment:
```suggestion
    command = f'"{python_exe}" "{python_script}"'```
I think this should work

Reply:
@SMoraisAnsys yes, agree. I'll fix all the occurences in this file, thanks!

Reply:
@gmalinve I added extra ``` at the end, sorry for that !!!!